golang/cosmos/app/upgrade.go

```diff
@@ -106,19 +99,14 @@ func buildProposalStepWithArgs(moduleName string, entrypoint string, opts map[st
 }
 
 func getVariantFromUpgradeName(upgradeName string) string {
-	switch upgradeName {
-	case "UNRELEASED_A3P_INTEGRATION":
-		return "A3P_INTEGRATION"
-	case "UNRELEASED_main":
-		return "MAINNET"
-	case "UNRELEASED_devnet":
-		return "DEVNET"
-		// Noupgrade for this version.
-	case "UNRELEASED_BASIC":
-		return ""
-	default:
-		return ""
-	}
+    switch upgradeName {
+    case "agoric-upgrade-18":
+        return "MAINNET"
+    case "agorictest-upgrade-18":
+        return "DEVNET"
+    default:
+        return ""
```

Comment:
Should we keep this or remove as we do in `isPrimaryUpgradeName`? To be fair, we would never reach this case as it will be short-circuited in `upgrade18Handler` before `getVariantFromUpgradeName` has a chance to get executed